jans-auth-server/server/src/main/java/io/jans/as/server/service/MDSUpdateTimer.java

```diff
@@ -0,0 +1,67 @@
+package io.jans.as.server.service;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import io.jans.fido2.service.mds.TocService;
+import io.jans.service.cdi.async.Asynchronous ;
+import io.jans.service.cdi.event;.Scheduled;
+import io.jans.service.timer.event;
+import io.jans.service.timer.schedule.TimerSchedule;
+import org.slf4j.Logger;
+
+/**
+ * @author madhumitas
+ *
+ */
+@ApplicationScoped
+@Named
+public class MDS3UpdateTimer {
+
+	private static final int DEFAULT_INTERVAL = 60 *60*24; // every 24 hours
+
+	@Inject
+	private Logger log;
+
+	@Inject
+	private Event<TimerEvent> timerEvent;
+
+	@Inject
+	private TocService tocService;
+
+	public void initTimer() {
```

Comment:
Formatting is messed. (Check other java files, e.g. spaces (vs tabs), curly brackets (starting bracket is always in same line))